azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -445,7 +448,18 @@ internal class CallingSDKWrapper(
             videoOptions?.let { joinCallOptions.outgoingVideoOptions = videoOptions }
             val callLocator: JoinMeetingLocator? = when (callConfig.callType) {
                 CallType.GROUP_CALL -> GroupCallLocator(callConfig.groupId)
-                CallType.TEAMS_MEETING -> TeamsMeetingLinkLocator(callConfig.meetingLink)
+                CallType.TEAMS_MEETING -> {
+                    if (!callConfig.meetingLink.isNullOrEmpty()) {
+                        TeamsMeetingLinkLocator(callConfig.meetingLink)
+                    } /* <MEETING_ID_LOCATOR> */ else if (!callConfig.meetingId.isNullOrEmpty() && !callConfig.meetingPasscode.isNullOrEmpty()) {
+                        TeamsMeetingIdLocator(callConfig.meetingId, callConfig.meetingPasscode)
+                    } /* </MEETING_ID_LOCATOR> */ else {
+                        throw CallCompositeException(
+                            "Teams Meeting information is incomplete",
+                            IllegalStateException()
+                        )
```

Comment:
Please check if we need to return error code call join failed, or may be handled in middleware?